entrypoint.sh

```diff
@@ -134,44 +88,35 @@ ListenPort = ${SUBSPACE_LISTENPORT}
 WGSERVER
 cat /data/wireguard/peers/*.conf >>/data/wireguard/server.conf
 
-if ip link show wg0 2>/dev/null; then
-  ip link del wg0
-fi
-ip link add wg0 type wireguard
-export SUBSPACE_IPV4_CIDR=$(echo ${SUBSPACE_IPV4_POOL-} | cut -d '/' -f2)
-ip addr add ${SUBSPACE_IPV4_GW}/${SUBSPACE_IPV4_CIDR} dev wg0
-export SUBSPACE_IPV6_CIDR=$(echo ${SUBSPACE_IPV6_POOL-} | cut -d '/' -f2)
-ip addr add ${SUBSPACE_IPV6_GW}/${SUBSPACE_IPV6_CIDR} dev wg0
-wg setconf wg0 /data/wireguard/server.conf
-ip link set wg0 up
-
-# dnsmasq service
-if ! test -d /etc/service/dnsmasq; then
-  cat <<DNSMASQ >/etc/dnsmasq.conf
-    # Only listen on necessary addresses.
-    listen-address=127.0.0.1,${SUBSPACE_IPV4_GW},${SUBSPACE_IPV6_GW}
+if test "${SUBSPACE_ENABLE_DNSMASQ}" -ne 0; then
+  # dnsmasq service
+  if ! test -d /etc/service/dnsmasq; then
+    cat <<DNSMASQ >/etc/dnsmasq.conf
+      # Only listen on necessary addresses.
+      listen-address=127.0.0.1
 
-    # Never forward plain names (without a dot or domain part)
-    domain-needed
+      # Never forward plain names (without a dot or domain part)
+      domain-needed
 
-    # Never forward addresses in the non-routed address spaces.
-    bogus-priv
+      # Never forward addresses in the non-routed address spaces.
+      bogus-priv
 DNSMASQ
 
-  mkdir -p /etc/service/dnsmasq
-  cat <<RUNIT >/etc/service/dnsmasq/run
-#!/bin/sh
-exec /usr/sbin/dnsmasq --no-daemon
+    mkdir -p /etc/service/dnsmasq
+    cat <<RUNIT >/etc/service/dnsmasq/run
+  #!/bin/sh
+  exec /usr/sbin/dnsmasq --no-daemon
 RUNIT
-  chmod +x /etc/service/dnsmasq/run
+    chmod +x /etc/service/dnsmasq/run
 
-  # dnsmasq service log
-  mkdir -p /etc/service/dnsmasq/log/main
-  cat <<RUNIT >/etc/service/dnsmasq/log/run
-#!/bin/sh
-exec svlogd -tt ./main
+    # dnsmasq service log
+    mkdir -p /etc/service/dnsmasq/log/main
+    cat <<RUNIT >/etc/service/dnsmasq/log/run
+  #!/bin/sh
+  exec svlogd -tt ./main
```

Comment:
please remove this change

Reply:
This is just an indentation. Thanks, I will remove.